packages/dd-trace/test/debugger/devtools_client/breakpoints.spec.js

```diff
@@ -463,8 +463,9 @@ describe('breakpoints', function () {
  * @param {object} [config] Optional configuration object.
  * @param {string} [config.id='probe-1'] The probe ID.
  * @param {number} [config.version=1] The probe version.
- * @param {object} [config.where={ sourceFile: 'test.js', lines: ['10'] }] The location information.
- * @param {object} [config.when={ json: { eq: [{ ref: 'foo' }, 42] }, dsl: 'foo = 42' }] The condition for the probe.
+ * @param {object} [config.where = { sourceFile: 'test.js', lines: ['10'] }] The location information.
+ * @param {object} [config.when] The condition for the probe.
+ *  { json: { eq: [{ ref: 'foo' }, 42] }, dsl: 'foo = 42' } by default.
```

Comment:
In all our other JSDoc comments where we wrap a line, we always indent by two spaces on the 2nd line:

```suggestion
 *   { json: { eq: [{ ref: 'foo' }, 42] }, dsl: 'foo = 42' } by default.
```